pylearn2/termination_criteria/__init__.py

```diff
@@ -221,16 +221,16 @@ def initialize(self, model):
         if self._new_epochs:
             self._epochs_done = 0
```

Comment:
Right, the behaviour is:
- `new_epochs=True`: start at 0, regardless of `transfer_experience`
- `transfer_experience=False`: start at 0, regardless of `new_epochs`
- `new_epochs=False` and `transfer_experience=True`: start at the previous number of epochs.
It makes sense to me at least, do you think it is confusing? Would you expect something different?